pkg/scheduler/reconciliation/persistent.go

```diff
@@ -383,9 +387,9 @@ func (r *PersistentReconciliationRepository) UpdateOperationState(schedulingID,
 			"SchedulingID":  schedulingID,
 			"State":         opStateOld, //ensure update will affect only operations which were not updated in between
 		}
-		cnt, err := q.Update().
-			Where(whereCond).
-			ExecCount()
+
+		update := q.Update().Where(whereCond)
+		cnt, err := update.ExecCount()
 
 		if cnt == 0 {
 			return fmt.Errorf("update of operation '%s' to state '%s' failed: no row was updated "+
```

Comment:
oh, we should probably check `err != nil` before to avoid that the 0-count isn't caused by a different issue